odhlasit.php

```diff
@@ -1,6 +1,5 @@
 <?php
 session_start();
-if (isset($_GET['odhlasit']) && $_GET['odhlasit'] == '1') {
-    $_SESSION['uzivatel'] = false;
-    $uzivatelPrihlaseny = false;
-}
\ No newline at end of file
+unset($_SESSION['uzivatel']);
+$_SESSION['status-prihlaseni'] = 401;
```

Comment:
Ja bych tam HTTP kody vubec netahal... 

Co je spis nice a i by se jim mohlo libit (i by na tom slo pak procvicit dalsi veci) jsou flash messages. Takze `$_SESSION["flash"][] = ['typ' => 'error', 'zprava' => "Nejde to"];` atd.